client/document/index.jsx

```diff
@@ -151,7 +168,7 @@ class Document extends React.Component {
 						} }
 					/>
 
-					{ i18nLocaleScript && <script src={ i18nLocaleScript } /> }
+					{ this.embedLocaleScript() }
```

Comment:
Let's inline a `<script />` tag here, and use a precomputed prop as `src`, see https://github.com/Automattic/wp-calypso/pull/26947/files#r213830884